shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -163,8 +164,9 @@ - (BOOL)isAccessibilityElement {
   // entire element tree looking for such a hit.
 
   //  We enforce in the framework that no other useful semantics are merged with these nodes.
-  if ([self node].HasFlag(blink::SemanticsFlags::kScopesRoute))
+  if ([self node].HasFlag(blink::SemanticsFlags::kScopesRoute) || [self node].platformViewId > -1)
```

Comment:
`>= kMinPlatformViewId` (is there a better name?) and define as constexpr above.

Reply:
Done, also added a function `IsPlatformViewNode()` in SemanticsNode to make it more readable.